test_util/cluster_api.py

```diff
@@ -187,7 +187,7 @@ def _authenticate(self):
         self.superuser_auth_cookie = r.cookies['dcos-acs-auth-cookie']
 
     def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
-                 dns_search_set, provider, auth_enabled, username, password):
+                 dns_search_set, provider, auth_enabled, username, password, default_os_user='root'):
```

Comment:
Why do you need to provide the value 'root' for `default_os_user` here and in `conftest.py`? Seems like only one should be necessary. Defining the same constant in 2 places is rarely necessary.